README.md

```diff
@@ -9,15 +9,14 @@ Python lib for common ontology operations over a variety of backends.
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.6456239.svg)](https://doi.org/10.5281/zenodo.6456239)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](.github/CODE_OF_CONDUCT.md) 
 
-This library provides a collection of different [interfaces](https://incatools.github.io/ontology-access-kit/interfaces/index.html) for different kinds of ontology operations, including:
+OAK provides a collection of [interfaces](https://incatools.github.io/ontology-access-kit/interfaces/index.html) for various ontology operations, including:
 
- - [lookup of basic features](https://incatools.github.io/ontology-access-kit/interfaces/basic.html) of an ontology element, such as it's label, definition, relationships, or aliases
+ - [look up basic features](https://incatools.github.io/ontology-access-kit/interfaces/basic.html) of an ontology element, such as its label, definition, relationships, or aliases
  - search an ontology for a term
- - validating an ontology
- - updating, deleting, or modifying terms
- - ontology term matching
```

Comment:
You dropped this? Why?

Reply:
I didn't drop those lines, I changed them for consistency! The changes show up as dropped lines and added lines.

Reply:
I cant find this specific line.. This seems to be removed without replacement.

Reply:
I removed "ontology term matching" after first changing it to "find matching terms" and concluding that was basically the same as "search an ontology for a term", but maybe there's something I'm missing.

Reply:
No no, very different: OAK has an ontology term matching module which results in a mapping table in SSSOM format! Different than search!

```
identify lexical matches and export as SSSOM mapping tables
```